soundrts/clientgame.py

```diff
@@ -336,7 +336,7 @@ def cmd_history_previous(self):
         voice.previous()
 
     def cmd_history_stop(self):
-        next(voice)
+        voice.next()
```

Comment:
What about next(...) calls in python3? I got errors prior about this because the object is not an iterator.